slack-api-model/src/main/java/com/slack/api/model/event/MessageMetadataUpdatedEvent.java

```diff
@@ -0,0 +1,25 @@
+package com.slack.api.model.event;
+
+import lombok.Data;
+
+import java.util.Map;
+
+/**
+ * https://api.slack.com/events/message_metadata_updated
```

Comment:
I noticed that there's some docs gaps for this and other event payloads https://api.slack.com/events?query=metad. I've put in a request to see if we can get that publicly available as well so that they're consistent with what they're releasing here.

Reply:
@srajiang Yeah, I also noticed that. This is not really great but this is not my first time to encounter this when implementing a new feature. I've made sure that the event data properties are correct, so that we can ship this change before the document improvements.